CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ### Added
+- [Dummy PR to trigger integ test flow]
```

Comment:
You dont need to do CHANGELOG for this :). We only need to do CHANGELOG for changes specific to js-sdk